cabal.project

```diff
@@ -0,0 +1,5 @@
+packages:
+  .
+  src/size-solver
+
+write-ghc-environment-files: always
```

Comment:
Is there a good reason for turning on ghc environment files? In my experience all they do is mess up your system.

Reply:
Yeah not sure.  Seems to work on CI.
I found that with late GHC 9 and v2-cabal, things rather break if you do _not_ have environment files.  Usually, packages are then not in scope...

Reply:
Do you have a concrete example of something that breaks if you remove it?